src/docs/release/breaking-changes/flutter-lints-package.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Introducing package:flutter_lints
+description: Migrate to package:flutter_lints to get the latest set of recommended lints, which encourage good coding practices.
+---
+
+## Summary
+
+The [`package:flutter_lints`][] defines the latest set of recommended lints that
+encourage good coding practices for Flutter apps, packages, and plugins. Projects
+created with `flutter create` after Flutter version 2.xx are already enabled to
+use the latest set of recommended lints. Projects created prior to that version
+can upgrade to it with the instructions in this guide.
+
+## Context
+
+Prior to the introduction of `package:flutter_lints`, the Flutter framework
+shipped with a set of lints defined in [`analysis_options_user.yaml`][] that was
+used by the [dart analyzer][] to identify code issues if a Flutter project
+didn't define a custom `analysis_options.yaml` file. Since `analysis_options_user.yaml`
+was tied to a particular framework version, it was difficult to evolve without
+breaking existing apps, packages, and plugins. As a result of that, the lints
+defined in `analysis_options_user.yaml` are heavily outdated. To fix this,
+`package:flutter_lints` was created. The package versions the lint set to enable
+evolving it without breaking existing projects. Since the package builds on
+Dart's [`package:lints`][] it also aligns the lints recommended for Flutter
+projects with the rest of the Dart universe.
+
+## Migration guide
+
+Follow these steps to migrate your Flutter project to use the latest recommended
+lints from `package:flutter_lints`:
+
+Add a dev_dependency on `package:flutter_lints` to your project's `pubspec.yaml`
+by running `flutter pub add --dev flutter_lints` in the root directory of the
+project.
+
+Create an `analysis_options.yaml` file in the root directory of your project
+(next to the `pubspec.yaml` file) with the following content:
+```yaml
+include: package:flutter_lints/flutter.yaml
+```
+
+The newly activated lint set may identify some previously undiscovered issues in
+your code. To find them, run `flutter analyze` in the root directory of your
```

Comment:
Perhaps also mention that you can simply open the project in an IDE with Dart support?